codecov_cli/services/upload/upload_collector.py

```diff
@@ -150,9 +150,9 @@ def generate_upload_data(self, report_type="coverage") -> UploadCollectionResult
             prep.run_preparation(self)
         logger.debug("Collecting relevant files")
         network = self.network_finder.find_files()
```

Comment:
Why are we not also ignoring files with the prefixes here?

Reply:
@joseph-sentry because this list will be the input into the overall coverage report. We want the prefixes here because this is what will let us know, for example to only look at files in a particularly subdirectory of a monorepo.